test/integration/testdata/zipkin.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: zipkin
+  labels:
```

Comment:
question: will this PR work if the user names the zipkin service as anything else? like istio-zipkin etc?